common/cont-lib/usr/share/cont-docs/70-general.txt.tpl

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run {% if privileged %} --privileged {% endif -%} {{ container }}{% if 'container-usage' not in cmd %} container-usage{% endif %}` to get this help.
+
+Run `docker run -ti {% if privileged %} --privileged {% endif -%} {{ container }} bash` to obtain interactive shell.
+
+Run `docker exec -ti {% if privileged %} --privileged {% endif -%} CONTAINERID bash` to access already running container.
```

Comment:
Good point, thats probably not needed for the basic usage described
in "General container help".